### PR TITLE
Docs: missing brackets in mutations docs

### PR DESCRIPTION
**Pull request by pollen8**
a couple of places where the objects were missing their closing braces

### CODE REVIEW

**Review by msutkowski — APPROVED** [review]

File: docs/concepts/mutations.md [hunk 1]
```diff
@@ -196,7 +196,7 @@ export const api = createApi({
           body,
         };
       },
-      invalidates: [{ type: 'Posts', id: 'LIST'],
+      invalidates: [{ type: 'Posts', id: 'LIST'}],
```
Comment (msutkowski) [inline docs/concepts/mutations.md:199]:
```suggestion
      invalidates: [{ type: 'Posts', id: 'LIST' }],
```

File: docs/concepts/mutations.md [hunk 2]
```diff
@@ -272,7 +272,7 @@ export const postApi = createApi({
       },
       // Invalidates all Post-type queries providing the `LIST` id - after all, depending of the sort order,
       // that newly created post could show up in any lists.
-      invalidates: [{ type: 'Posts', id: 'LIST'],
+      invalidates: [{ type: 'Posts', id: 'LIST'}],
```
Comment (msutkowski) [inline docs/concepts/mutations.md:275]:
```suggestion
      invalidates: [{ type: 'Posts', id: 'LIST' }],
```

**Review by msutkowski — CHANGES_REQUESTED** [review]
Thanks for catching this @pollen8. Mind updating the spacing here?

File: docs/concepts/mutations.md (hunk 1 above)
Comment (pollen8) [reply docs/concepts/mutations.md]: oh lol I missed the "commit suggestion" button, but I think I've updated anyway!

**Review by msutkowski — APPROVED** [review]

**Pull request merged**